kernel/kernel-api/src/main/java/io/delta/kernel/internal/snapshot/SnapshotManager.java

```diff
@@ -534,8 +536,6 @@ private SnapshotImpl createSnapshot(LogSegment initSegment, Engine engine) {
             initSegment,
             Optional.ofNullable(latestSnapshotHint.get()));
 
-    long startTimeMillis = System.currentTimeMillis();
```

Comment:
btw, it would have been more accurate to move the logger.info here, too. Now we are also including `assertLogFilesBelongToTable` in our time calculation

Reply:
@scottsand-db good point, please see this pull request https://github.com/delta-io/delta/pull/3974